WordPress/Classes/ViewRelated/NUX/SigninEditingState.swift

```diff
@@ -2,5 +2,5 @@ import Foundation
 
 class SigninEditingState {
     static var signinEditingStateActive = false
-    static var signinLastKeyboardHeightDelta = CGFloat(0.0)
+    static var signinLastKeyboardHeightDelta: CGFloat = 0
```

Comment:
Suprisingly yes. I guess `CGFloat` has 15 different constructors, and the compiler has to figure out which one it's the right one, while in the second case, there's an explicit type and a literal.